### PR TITLE
[CARBONDATA-3791] update pycarbon document

### PR DESCRIPTION
**Pull request by ajantha-bhat**
### Why is this PR needed?
 setup.cfg lies outside the folder. so need to update the document on the same.

 
 ### What changes were proposed in this PR?
 setup.cfg lies outside the folder. so need to update the document on the same.
some other minor fix.
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

**Comment by ajantha-bhat** [comment]
@xubo245 @jackylk @kunal642 : please check

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1312/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/3031/

File: python/setup.py [hunk 1]
```diff
@@ -44,7 +44,6 @@
     'six>=1.5.0',
     'torchvision>=0.2.1',
     'tensorflow>=1.4.0',
-    'jnius>=1.1.0',
```
Comment (xubo245) [inline python/setup.py:47]: windows need jnius, please check it

File: python/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ PyCarbon provides python API for integrating CarbonData with AI framework like
 
 $ git clone https://github.com/apache/carbondata.git
 
-$ cd python/pycarbon
```
Comment (xubo245) [inline python/README.md:9]: suggesion:  add pycarbon fold to line 17 and other place

Comment (ajantha-bhat) [reply python/README.md:9]: done

File: python/setup.py (hunk 1 above)
Comment (ajantha-bhat) [reply python/setup.py:47]: pyjnius doesn't work in windows ?

Comment (ajantha-bhat) [reply python/setup.py]: ok. I reverted.

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1315/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/3034/

**Comment by kunal642** [comment]
LGTM

**Pull request closed without merging**